packages/dashboard/test/dashboard-section.test.ts

```diff
@@ -112,4 +117,97 @@ describe('dashboard section', () => {
       expect(getMoveBackwardButton(section)?.getAttribute('title')).to.eql('baz');
     });
   });
+
+  describe('title heading level', () => {
```

Comment:
I'd suggest to merge this with the tests in `dashboard-layout.test.js`, also the first two tests seem duplicates.